org.gnu.emacs.json

```diff
@@ -17,17 +17,15 @@
     ],
     "modules": [
         {
-            "name": "emacs",
-            "sources": [
-                {
-                    "type": "archive",
-                    "url": "http://www.nic.funet.fi/pub/gnu/ftp.gnu.org/pub/gnu/emacs/emacs-25.3.tar.xz",
-                    "sha256": "253ac5e7075e594549b83fd9ec116a9dc37294d415e2f21f8ee109829307c00b"
-                },
-                {
-                    "type": "patch",
-                    "path": "locale-fixup.patch"
-                },
+	    "name": "emacs",
```

Comment:
These added lines should use spaced for indentation, as is done in the rest of the file.

Reply:
The complaint was that it was pulled from a random mirror, if we are going to keep the tarball can we at least use the savanna tarball?

Reply:
Absolutely would be better to use a less "random" mirror. The one we have for 25.3 is was an oversight on my part, but it does not really prevent anyone from verifying the source.

For 26.1 we I think might use the link from release announcement mail, http://ftpmirror.gnu.org/emacs/emacs-26.1.tar.xz